homeassistant/components/schlage/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""The Schlage integration."""
+from __future__ import annotations
+
+import pyschlage
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.LOCK]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Schlage from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+    auth = await hass.async_add_executor_job(
```

Comment:
Please handle transient failures by raising ConfigEntryNotReady so it tries to setup later

Reply:
Done.